### ISSUE TITLE
The question about the temperature of loss in MocoV3

### ISSUE THREAD

**Issue by Lin-Qingyang-Alec** [facebookresearch/moco-v3#31]
I see the loss in MocoV2 just like:
     loss = nn.CrossEntropyLoss()(logits / self.T, labels)
but the loss in MocoV3 just like:
     loss = nn.CrossEntropyLoss()(logits / self.T, labels) * (2 * self.T)

I don't know why the loss should be multiplied by 2*temperature, it really confuses me. If anyone can answer my confusion.

**Comment by realcy** [comment]
I have the same question, did you find out why?
